src/components/AddAssignment.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { Link } from 'react-router-dom'
+import Button from '@mui/material/Button';
+import { TextField } from '@mui/material';
+
+class AddAssignment extends React.Component{
+
+    constructor(props) {
+        super(props)
+        this.state={nameAssignment: '', dueDate: '', course: ''};
+    }
+    // this is called when the user pushes the button to add an assignment
+    handleAssignment = () => {
+        // this is the REST API that adds an assignment
+        fetch('http://localhost:8081/addAssignment',
```

Comment:
can this fetch call not use the SERVER_URL? perhaps its a difference in original repository.